inference-engine/src/gna_plugin/descriptions/gna_input_desc.hpp

```diff
@@ -18,6 +18,7 @@ struct InputDesc {
     std::unordered_map<std::string, intel_dnn_orientation_t> orientation_in;
     /// order of scale factors matches inputs order in original topology
     std::vector<float> inputScaleFactors;
+    std::vector<uint8_t> inputPrecisions;
```

Comment:
I'm wondering whether what we would store after moving to ngraph? Which numbers?